accounts/abi/bind/base.go

```diff
@@ -49,6 +53,10 @@ type TransactOpts struct {
 	Nonce  *big.Int       // Nonce to use for the transaction execution (nil = use pending state)
 	Signer SignerFn       // Method to use for signing the transaction (mandatory)
 
+	TransactionManagerURL string   // Tessera/Constellation URL to call /storeraw API.
```

Comment:
`ethclient.Client` struct defines wrappers for Ethereum RPC API, it's the right place to define wrappers for Private Transaction Manager public API.

Reply:
This is quite tricky. It seems that we cannot create transaction in `ethclient.Client` if we want to use abigen go binding code to create transaction.

Reply:
`ethclient.Client` implements `ContractBackend` interface and hence it is the implementation of `ContractTransactor` interface.

Reply:
Adding `TransactionManagerURL` as a field in `ethclient.Client` struct